src/utils/changelog.ts

```diff
@@ -855,7 +910,12 @@ export async function generateChangesetFromGit(
     }
   }
 
-  return changelogSections.join('\n\n');
+  return {
+    changelog: changelogSections.join('\n\n'),
+    bumpType,
+    totalCommits: gitCommits.length,
+    matchedCommitsWithSemver: foundBumpTypes.size,
```

Comment:
This value dosen't seem to be correct as `foundBumpTypes` can only be 3 max (there are only 3 bump types and foundBumpTypes is a set).